src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,13 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        User result = null;
+        for (int i = 0; i < users.length; i++) {
+            if (email.equals(users[i].getEmail())) {
+                result = users[i];
+                break;
+            }
+        }
```

Comment:
Feel free to use _for-each_ loop here. Also, you can simply return result when you find it rather than `break` the loop.

Reply:
Thanks for remark! It will be more convenient and easier than my code.